ldes-server-application/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>be.vlaanderen.informatievlaanderen.vsds</groupId>
         <artifactId>ldes-server</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
+        <version>0.0.2-SNAPSHOT</version>
```

Comment:
Based on what are we already going to version 0.0.2? In the end, it's just a number, but since we've presented nothing yet, I find it odd to upgrade already